sdk/src/clearingHouseUser.ts

```diff
@@ -1395,11 +1395,7 @@ export class ClearingHouseUser {
 	 * @returns feeForQuote : Precision QUOTE_PRECISION
 	 */
 	public calculateFeeForQuoteAmount(quoteAmount: BN): BN {
-		const feeStructure = this.clearingHouse.getStateAccount().perpFeeStructure;
-
-		return quoteAmount
-			.mul(feeStructure.feeNumerator)
-			.div(feeStructure.feeDenominator);
+		return quoteAmount.mul(new BN(10)).div(new BN(10000));
```

Comment:
todo

Reply:
yeah we'll need to hard code it in the sdk 😓